### ISSUE TITLE
Modify the loss function and Optimizer

### ISSUE THREAD

**Issue by Nandhan-Golla** [GDG-Open-Challenge/AI-ML-2--Sign-Language-Detection-CNN#3]
The current script uses SGD Optimizer with categorical crossentropy. Better functions can be used than that. 
Benchmark with different functions with hyperparameter tuning (Cross-Fold or K-Fold). 

The result must return better outputs than the previous variants.
